apmpackage/apm/changelog.yml

```diff
@@ -1,6 +1,11 @@
 # newer versions go on top
 #
 # change type can be one of: enhancement, bugfix, breaking-change
+- version: "7.16.1"
```

Comment:
elastic-package doesn't like forward looking versions here.

Reply:
ugh, looks like we need to wait with merging then; that's not great though.